packages/ebayui-core/src/components/ebay-date-textbox/component.ts

```diff
@@ -16,28 +16,15 @@ export interface InvalidDateEvent {
     index: number;
 }
 
-interface DateTextboxInput {
+interface DateTextboxInput extends Omit<CalendarInput, `on${string}`> {
```

Comment:
Most of these types can be removed if we extend calendar, since they are directly piped through